src/components/game/GamePage.js

```diff
@@ -28,11 +28,11 @@ export default function GamePage() {
   const [blueTeamWord, setBlueTeamWord] = useState('')
   const [whiteTeamWord, setWhiteTeamWord] = useState('')
   const [role, setRole] = useState('')
-  const [screenshotTime, setScreenshotTime] = useState()
   const [players, setPlayers] = useState()
   const [winningTeam, setWinningTeam] = useState()
   const [winningJudge, setWinningJudge] = useState()
-  const [screenshotTimer, setScreenshotTimer] = useState()
+  // const [goNextRound, setGoNextRound] = useState(false)
+  const [isScreenshotPage, setIsScreenshotPage] = useState(false)
 
   // const setRole = async role => {}
```

Comment:
sorry this comment was me, I should've removed this before but can you remove it here